pos_main.py

```diff
@@ -0,0 +1,162 @@
+"""This script facilitates plus one scaling for disperse volume"""
+
+from scale.gf_plus_one_scale import *
+from test_data.gf_plus_one_test_data import *
+from scale.gf_logs import *
+from scale.gf_exceptions import *
+from scale.gf_api import *
+import os
+
+
+def gf_get_volume_and_bricks():
+    """Get volume name, new node and file name from user.
+    """
+    volname = input("Enter name of the volume: ")
+    filename = input("Enter file name which contains new bricks: ")
+    hostname = input("Enter IP/hostname of new node: ")
+
+    if not volname or not filename or not hostname:
+        glogger.debug('Invalid volume name and filename')
```

Comment:
Incomplete detail in log message. Even if you see the message in log you can't make out why it is wrong. Add input values in log.